src/components/ProfileStats.vue

```diff
@@ -1,23 +1,89 @@
 <template>
-  <div class="max-w-sm m-auto flex">
+  <div class="max-w-sm m-auto flex bg-white">
+    <UpdateModal v-model="modalOpen">
+      <div class="text-gray-700 center-text">
+        <h3 class="text-4xl pb-6">Update Shipping Address</h3>
+        <form @submit.prevent="updateShipping">
```

Comment:
This is the update shipping modal html, update modal is designed to accept any html it is passed through the use of <slot />